packages/client/src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,142 @@
+import { DishTagsEnum, MealTypeEnum } from '@fullstack/sdk';
+import ClearIcon from '@mui/icons-material/Clear';
+import { Box, Button, Stack } from '@mui/material';
+import Checkbox from '@mui/material/Checkbox';
+import FormControl from '@mui/material/FormControl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormGroup from '@mui/material/FormGroup';
+import FormLabel from '@mui/material/FormLabel';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import _ from 'lodash';
+import { ChangeEvent, useEffect, useState } from 'react';
+
+type CheckboxProps = {
+  listLabel: string;
+  filtersName: string;
+  filters: ObjType;
+  parentCallback: { name: string | null; value: string | null }[];
```

Comment:
Chyba nazwa callback jest trochę mylna? Biorąc pod uwagę, że to obiekt, więc nigdzie się tego nie wywołuje ("call"-uje)

Reply:
racja, już poprawiam